ktlint-ruleset-experimental/src/main/kotlin/com/pinterest/ktlint/ruleset/experimental/AnnotationRule.kt

```diff
@@ -30,6 +37,7 @@ class AnnotationRule : Rule("annotation") {
             "Multiple annotations should not be placed on the same line as the annotated construct"
         const val annotationsWithParametersAreNotOnSeparateLinesErrorMessage =
             "Annotations with parameters should all be placed on separate lines prior to the annotated construct"
+        const val fileAnnotationsShouldBeSeparated = "File annotations should be separated from packages with a blank line"
```

Comment:
Following up from #747, I think the wording should be a bit different, cause there's not necessarily a package coming after a file annotation (e.g. in `.kts` files). So maybe change it to `from file contents` or something like that.